quiz-template.html

```diff
@@ -1,148 +1,185 @@
+              case "x": // category x
+                xTotal++;
+                break;
+              case "y": // category y
+                yTotal++;
+                break;
+              case "z": //category z
+                zTotal++;
+                break;
             }
-		}
-	}
-}
-
-/* Generates the text to appear in the results */
-function generateCategory(){
-	if(xTotal > yTotal && xTotal > zTotal){ // category x
-		categoryName = "X";
-		description = "Supplementary information for x :3"
-	} else if (yTotal > xTotal && yTotal > zTotal){ // category y
-		categoryName = "Y";
-		description = "Supplementary information for y :3"
-	} else if (zTotal > xTotal && zTotal > yTotal){ // catgory z
-		categoryName = "Z";
-		description = "Supplementary information for z :3"
-	} else { // if ties are possible or if no questions are answered
-		categoryName = "Any"
-		description = "Hm maybe you should retake the quiz? :3"	
-	}
-}
-
-/* TODO: 
-- Check whether the user answered all questions.
-- Create a button to reset the quiz. 
-*/
-function displayResult(){
-	grade();
-	generateCategory();
-	hideHome();
-	document.write("<font face = 'Arial'><center>Congratulations! You are in the </font>" + "<br><font size = '14px'>" + categoryName + "</font><br>" + "<font face = 'Arial'>category!" + "<br>" + description + "</center></font>");
-}
-</script>
-
-<body bgcolor="#FFFFFF">
-
-<!-- Home -->
-<div id = "home">
-
-<!-- Form -->
-<form method="POST" name="quiz">
-
-<p><font face="Arial"><big><big>Quiz Title</big></big></font></p>
-
-<!-- Questions. NOTE: To avoid ties, choose 4, 7, 10, etc. questions -->
-<div class="questions">
-Question #1
-</div>
-
-<div class="answers">
-<input type ="radio" value ="x" name="question1"> a) Description <br>
-<input type ="radio" value ="y" name="question1"> b) Description <br>
-<input type ="radio" value ="z" name="question1"> c) Description <br>
-
-</div>
-
-<br>
-
-<div class="questions">
-Question #2
-</div>
-
-<div class="answers">
-<input type ="radio" value ="x" name="question2"> a) Description <br> 
-<input type ="radio" value ="y" name="question2"> b) Description <br>
-<input type ="radio" value ="z" name="question2"> c) Description <br>
-</div>
-
-<br>
-
-<div class="questions"">
-Question #3
-</div>
-
-<div class="answers">
-<input type ="radio" value ="x" name="question3"> a) Description <br>
-<input type ="radio" value ="y" name="question3"> b) Description <br>
-<input type ="radio" value ="z" name="question3"> c) Description <br> 
-</div>
-
-<br>
-
-</form>
-
-<!-- Submit button -->
-<form>
-<input type = "button" value="Submit!" name ="submit" onClick="displayResult()">
-</form>
-
-</div> 
+          }
+        }
+      }
+
+      /* Generates the text to appear in the results */
+      function generateCategory() {
+        if (xTotal > Math.max(yTotal, zTotal)) { // category x
+          categoryName = "X";
+          categoryDescription = "Supplementary information for x :3";

```

Comment:
These can probably be better implemented as arrays or objects so that they'd be easier to access (e.g. via a loop or directly as the index), but for now, I'll leave it.